### ISSUE TITLE
стояла база, скрипт подтянул но в итоге все пусто

### ISSUE THREAD

**Issue by de-salvatierra** [craft-soft/csbans2#6]
стояла база, скрипт подтянул но в итоге все пусто
+проблема с текстом

![image](https://github.com/craft-soft/csbans2/assets/57546436/05aa1554-40b6-4eea-afe2-28fa9b4eba55)
![image](https://github.com/craft-soft/csbans2/assets/57546436/66cc39dc-a494-42e0-b68d-f5e1c5f7742d)

_Originally posted by @iwillnotleave in https://github.com/craft-soft/csbans2/issues/3#issuecomment-1722480044_

**Comment by de-salvatierra** [comment]
В данной реализации нет обновлятора, и не будет работать на старых базах, только чистая установка. В ближайшее время сделаю обновлятор

**Comment by DeeNiceGray** [comment]
> В данной реализации нет обновлятора, и не будет работать на старых базах, только чистая установка. В ближайшее время сделаю обновлятор

Нет, данная проблема, отсутствия текстов на значения, сохраняется и при установке вместе с базой.
